core/src/main/java/io/methvin/watcher/DirectoryWatcher.java

```diff
@@ -59,28 +59,35 @@
 
   // this is set to true/false depending on whether recursive watching is supported natively
   private Boolean fileTreeSupported = null;
+  private boolean enableFileHashing = true;
 
   public static DirectoryWatcher create(Path path, DirectoryChangeListener listener) throws IOException {
-    return create(Collections.singletonList(path), listener);
+    return create(Collections.singletonList(path), listener, Boolean.TRUE);
```

Comment:
Basically everywhere you use `Boolean` instances here you can use `boolean`, so this can be `true`.